src/instruments/src/RTPI/index.tsx

```diff
@@ -11,8 +11,8 @@ const RTPIDisplay = () => {
 
     return (
         <>
-            <text x="0" y="115" className="direction">{ltsTest === 0 ? '8' : directionText}</text>
-            <text x="330" y="115" className="value">{ltsTest === 0 ? '88.8' : `${Math.abs(rudderTrim).toFixed(1)}`}</text>
+            <text x="0" y="110" className="direction">{ltsTest === 0 ? '8' : directionText}</text>
```

Comment:
The `8` character needs to be changed to a combined `L` and `R` for the lights test.

Reply:
> The `8` character in the font needs to be changed to a combined `L` and `R` for the lights test.

I think it would only affect the direction indicator and not the numbers itself, thinking about an extra char in the font which would be the combined L/R, rather than changing the 8 itself

Reply:
![image](https://user-images.githubusercontent.com/29005005/120710145-69fcf700-c4db-11eb-846e-4dc6368f8018.png)

Fixed with test char set to "T"